{"protocolVersion":"0.1.1","runnerVersion":null,"pid":3508,"type":"start","time":0}
{"suite":{"id":0,"platform":"vm","path":"/Users/quocnguyen/Documents/FlutterExample/my_app/test/widget_test.dart"},"type":"suite","time":0}
{"test":{"id":1,"name":"loading /Users/quocnguyen/Documents/FlutterExample/my_app/test/widget_test.dart","suiteID":0,"groupIDs":[],"metadata":{"skip":false,"skipReason":null},"line":null,"column":null,"url":null},"type":"testStart","time":1}
{"count":1,"type":"allSuites","time":3}
{"testID":1,"result":"success","skipped":false,"hidden":true,"type":"testDone","time":1500}
{"group":{"id":2,"suiteID":0,"parentID":null,"name":"","metadata":{"skip":false,"skipReason":null},"testCount":1,"line":null,"column":null,"url":null},"type":"group","time":1504}
{"test":{"id":3,"name":"Counter increments smoke test","suiteID":0,"groupIDs":[2],"metadata":{"skip":false,"skipReason":null},"line":126,"column":5,"url":"package:flutter_test/src/widget_tester.dart","root_line":14,"root_column":3,"root_url":"file:///Users/quocnguyen/Documents/FlutterExample/my_app/test/widget_test.dart"},"type":"testStart","time":1505}
{"testID":3,"messageType":"print","message":"══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════\nThe following StateError was thrown running a test:\nBad state: No element\n\nWhen the exception was thrown, this was the stack:\n#0      Iterable.single (dart:core/iterable.dart:558:25)\n#1      WidgetController._getElementPoint (package:flutter_test/src/controller.dart:814:47)\n#2      WidgetController.getCenter (package:flutter_test/src/controller.dart:786:12)\n#3      WidgetController.tap (package:flutter_test/src/controller.dart:257:18)\n#4      main.<anonymous closure> (file:///Users/quocnguyen/Documents/FlutterExample/my_app/test/widget_test.dart:23:18)\n<asynchronous suspension>\n#5      main.<anonymous closure> (file:///Users/quocnguyen/Documents/FlutterExample/my_app/test/widget_test.dart)\n#6      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:144:29)\n<asynchronous suspension>\n#7      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart)\n#8      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:794:19)\n<asynchronous suspension>\n#11     TestWidgetsFlutterBinding._runTest (package:flutter_test/src/binding.dart:774:14)\n#12     AutomatedTestWidgetsFlutterBinding.runTest.<anonymous closure> (package:flutter_test/src/binding.dart:1191:24)\n#13     FakeAsync.run.<anonymous closure>.<anonymous closure> (package:fake_async/fake_async.dart:178:54)\n#18     withClock (package:clock/src/default.dart:48:10)\n#19     FakeAsync.run.<anonymous closure> (package:fake_async/fake_async.dart:178:22)\n#24     FakeAsync.run (package:fake_async/fake_async.dart:178:7)\n#25     AutomatedTestWidgetsFlutterBinding.runTest (package:flutter_test/src/binding.dart:1188:15)\n#26     testWidgets.<anonymous closure> (package:flutter_test/src/widget_tester.dart:136:24)\n#27     Declarer.test.<anonymous closure>.<anonymous closure> (package:test_api/src/backend/declarer.dart:175:19)\n<asynchronous suspension>\n#28     Declarer.test.<anonymous closure>.<anonymous closure> (package:test_api/src/backend/declarer.dart)\n#33     Declarer.test.<anonymous closure> (package:test_api/src/backend/declarer.dart:173:13)\n#34     Invoker.waitForOutstandingCallbacks.<anonymous closure> (package:test_api/src/backend/invoker.dart:231:15)\n#39     Invoker.waitForOutstandingCallbacks (package:test_api/src/backend/invoker.dart:228:5)\n#40     Invoker._onRun.<anonymous closure>.<anonymous closure>.<anonymous closure> (package:test_api/src/backend/invoker.dart:383:17)\n<asynchronous suspension>\n#41     Invoker._onRun.<anonymous closure>.<anonymous closure>.<anonymous closure> (package:test_api/src/backend/invoker.dart)\n#46     Invoker._onRun.<anonymous closure>.<anonymous closure> (package:test_api/src/backend/invoker.dart:370:9)\n#47     Invoker._guardIfGuarded (package:test_api/src/backend/invoker.dart:415:15)\n#48     Invoker._onRun.<anonymous closure> (package:test_api/src/backend/invoker.dart:369:7)\n#55     Invoker._onRun (package:test_api/src/backend/invoker.dart:368:11)\n#56     LiveTestController.run (package:test_api/src/backend/live_test_controller.dart:153:11)\n#57     RemoteListener._runLiveTest.<anonymous closure> (package:test_api/src/remote_listener.dart:256:16)\n#62     RemoteListener._runLiveTest (package:test_api/src/remote_listener.dart:255:5)\n#63     RemoteListener._serializeTest.<anonymous closure> (package:test_api/src/remote_listener.dart:208:7)\n#81     _GuaranteeSink.add (package:stream_channel/src/guarantee_channel.dart:125:12)\n#82     new _MultiChannel.<anonymous closure> (package:stream_channel/src/multi_channel.dart:159:31)\n#86     CastStreamSubscription._onData (dart:_internal/async_cast.dart:85:11)\n#120    new _WebSocketImpl._fromSocket.<anonymous closure> (dart:_http/websocket_impl.dart:1145:21)\n#128    _WebSocketProtocolTransformer._messageFrameEnd (dart:_http/websocket_impl.dart:338:23)\n#129    _WebSocketProtocolTransformer.add (dart:_http/websocket_impl.dart:232:46)\n#139    _Socket._onData (dart:io-patch/socket_patch.dart:1969:41)\n#148    new _RawSocket.<anonymous closure> (dart:io-patch/socket_patch.dart:1505:33)\n#149    _NativeSocket.issueReadEvent.issue (dart:io-patch/socket_patch.dart:1017:14)\n(elided 111 frames from dart:async and package:stack_trace)\n\nThe test description was:\n  Counter increments smoke test\n════════════════════════════════════════════════════════════════════════════════════════════════════","type":"print","time":2098}
{"testID":3,"error":"Test failed. See exception logs above.\nThe test description was: Counter increments smoke test","stackTrace":"","isFailure":false,"type":"error","time":2101}
{"testID":3,"result":"error","skipped":false,"hidden":false,"type":"testDone","time":2102}
{"success":false,"type":"done","time":2124}
